index.bs

```diff
@@ -853,6 +846,32 @@ The dictionary SHALL have the following fields:
 : <dfn for="StreamAbortInfo" dict-member>errorCode</dfn>
 :: The error code. The default value of 0 means "CLOSING."
 
+## Procedures ##  {#outgoing-stream-procedures}
+
+When an {{OutgoingStream}} receives a STOP_SENDING frame from the remote side,
+the user agent MUST queue a task to run the following:
+      1. Let |stream| be the {{OutgoingStream}} object.
+      1. Let |transport| be the {{WebTransport}}, which the |stream| was created
+         from.
+      1. Remove |stream| from the |transport|'s {{[[OutgoingStreams]]}}
+         internal slot.
```

Comment:
Let's remove "internal slot" as discussed at #223.

Reply:
@yutakahirano Sorry, the rebase was quite painful and "internal slot" must have snuck back in. I like the shorter format. I'll remember to squash next time to avoid bugs like this.